crates/rustc_codegen_c/src/lib.rs

```diff
@@ -88,6 +96,10 @@ impl CodegenBackend for CCodegen {
     ) -> Result<(), ErrorGuaranteed> {
         link_binary(sess, &crate::archive::ArArchiveBuilderBuilder, &codegen_results, outputs)
     }
+
+    fn supports_parallel(&self) -> bool {
+        false // Maybe true?
```

Comment:
You can probably just drop this comment, I don't think there is any point in worrying about parallel codegen at this point.